pkg/util/names/names_test.go

```diff
@@ -76,3 +76,27 @@ func TestGetClusterName(t *testing.T) {
 		})
 	}
 }
+
+func TestGenerateBindingReferenceKey(t *testing.T) {
+	tests := []struct {
+		name      string
+		namespace string
```

Comment:
```suggestion
		want string
```
We can add a `want` field here.

Reply:
Sorry, not replace `namespace` with `want`, but add a new field.